types/saml20/index.d.ts

```diff
@@ -0,0 +1,30 @@
+// Type definitions for saml20 0.1
+// Project: https://github.com/leandrob/saml20
+// Definitions by: Jan Petzold <https://github.com/HackerUndKoch>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+export interface Profile {
+    issuer: string;
+    claims: any;
+    audience?: string;
+    sessionIndex?: string; // only SAML 2.0
+}
+
+export type ValidationOptions = ValidationOptionsWithKey | ValidationOptionsWithPrint;
+
+export interface ValidationOptionsWithKey {
+    publicKey: string;
+    bypassExpiration?: boolean;
+    audience?: any;
+}
+
+export interface ValidationOptionsWithPrint {
+    thumbprint: string;
+    bypassExpiration?: boolean;
+    audience?: any;
+}
+
+export type ParseCallback = (err: Error, profile: Profile, version: string) => void;
```

Comment:
I changed it now to `Error | null` since the original library always uses `null` if there is no error.
Thanks for your time.